packages/@react-spectrum/card/docs/CardView.mdx

```diff
@@ -0,0 +1,509 @@
+  {id: 2, name: 'Ivysaur', type: 'Grass, Poison', width: 475, height: 475, src: 'https://assets.pokemon.com/assets/cms2/img/pokedex/full/002.png'},
+  {id: 3, name: 'Venusaur', type: 'Grass, Poison', width: 475, height: 475, src: 'https://assets.pokemon.com/assets/cms2/img/pokedex/full/003.png'},
+  {id: 4, name: 'Charmander', type: 'Fire', width: 475, height: 475, src: 'https://assets.pokemon.com/assets/cms2/img/pokedex/full/004.png'},
+  {id: 5, name: 'Charmeleon', type: 'Fire', width: 475, height: 475, src: 'https://assets.pokemon.com/assets/cms2/img/pokedex/full/005.png'},
+  {id: 6, name: 'Charizard', type: 'Fire, Flying', width: 475, height: 475, src: 'https://assets.pokemon.com/assets/cms2/img/pokedex/full/006.png'}
+  ///- end collapse -///
+];
+
+function CardViewActions() {
+  let [selectedKeys, setSelectedKeys] = React.useState(new Set([2]));
+
+  return (
+    <ActionBarContainer height="500px">
+      <CardView items={items} selectedKeys={selectedKeys} onSelectionChange={setSelectedKeys} height="500px" maxWidth="800px" layout={GridLayout} aria-label="CardView Pokemon with ActionBar example" selectionMode="single">
+        {item => (
+          <Card textValue={item.name} width={item.width} height={item.height}>
+            <Image src={item.src} />
+            <Heading>{item.name}</Heading>
+            <Text slot="detail">PNG</Text>
+            <Content>{item.type}</Content>
+          </Card>
+        )}
+      </CardView>
+      <ActionBar
+        selectedItemCount={selectedKeys.size}
+        onClearSelection={() => {
+          setSelectedKeys(new Set());
+        }}
+        onAction={key => alert(`Performing ${key} action on ${selectedKeys.keys().next().value}`)}>
+        <Item key="edit">
+          <Edit />
+          <Text>Edit</Text>
+        </Item>
+        <Item key="copy">
+          <Copy />
+          <Text>Copy</Text>
+        </Item>
+        <Item key="delete">
+          <Delete />
+          <Text>Delete</Text>
+        </Item>
+      </ActionBar>
+    </ActionBarContainer>
+  );
+}
+
+<CardViewActions />
+```
+
+## Props
+
+### CardView props
+
+<PropTable component={docs.exports.CardView} links={docs.links} style={{marginBottom: '40px'}} />
+
+### Card props
+
+<PropTable component={{
+  props: {
+    properties: Object.fromEntries(Object.entries(cardViewTypes.exports.CardViewCardProps.properties))
+  }
+}} links={cardViewTypes.links} />
+
+## Visual options
+
+### Horizontal Cards
+GridLayout is the only layout that currently supports horizontal cards. To enable horizontal cards, provide `cardOrientation="horizontal"`
+to the CardView and to your GridLayout instance if you are instantiating one.
+
+```tsx example
+// Same CardView from GridLayout example...
+<CustomLayout layout={GridLayout} aria-label="CardView with horizontal cards" height="400px" cardOrientation="horizontal" />
+```
```

Comment:
Example looks a bit broken but is fixed by css changes in https://github.com/adobe/react-spectrum/pull/2737